src/seeds/index.ts

```diff
@@ -62,10 +62,10 @@ const seedModel = async (seedData: Record<string, any[]>) => {
 				}
 				if (model === "restaurant") {
 					for (const restaurant of rows as Prisma.RestaurantCreateInput[]) {
-						const { admin, ...restautantUpdateInput } = restaurant;
+						const { admin, ...RestaurantUpdateInput } = restaurant;
```

Comment:
El nombre de las variables siempre en camel case por fa jeje

```suggestion
						const { admin, ...restaurantUpdateInput } = restaurant;
```